src/index.rs

```diff
@@ -356,13 +356,16 @@ impl Index {
 
     /// imports the corresponding TypeSize (according to the given initializer-id) from the given ConstExpressions
     /// into self's const-expressions and returns the new Id
+    ///
+    /// panics if the import fails (e.g. the given TypeSice::ConstExpression(id) does not exist in this Index)
```

Comment:
Typo : TypeSize

Reply:
I appreciate you reading even the comments carefully 👍 
done